codegen/deploy.sh

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env bash
-
-./gradlew clean :deployJar
```

Comment:
Are you intending to type the gradle command directly?

Reply:
Yesterday, when you said to delete the gradlew files, I misunderstood that you would not use the gradlew command so I deleted this files, and at the same time I deleted the deploy.sh file. Do you want me to revert these files only in the codegen or all? Sorry for my misunderstanding

Reply:
I want you to delete the automatically generated files when we run the `gradle wrapper` command. It's because we will run the command in the new environment and don't want to commit this file. It make us to be confused to make a PR that has non-related files.

Reply:
I will do that